README.md

```diff
@@ -138,4 +139,49 @@ curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d \
     }
   }
 }'
-```
\ No newline at end of file
+```
+
+
+## Building locally
+The Hypertrace GraphQl service uses gradlew to compile/install/distribute. Gradle wrapper is already part of the source code. To build Hypertrace GraphQL image, run:
+
+```
+./gradlew dockerBuildImages
+```
+
+## Testing
+
+### Running unit tests
+Run `./gradlew test` to execute unit tests. 
+
+### Testing image
+
+To test your image using the docker-compose setup follow the steps:
+
+- Commit you changes to a branch say `graphql-service-test`.
```

Comment:
nit: We describe how to test via docker compose in a section called "Testing image" and via helm in one called "Helm setup". I think It would make more sense to break it down into two sections under "Testing image" like "With docker-compose" and "With helm"